SECURITY.md

```diff
@@ -1,5 +1,7 @@
 # Reporting a Vulnerability
 
-**If you've found a security vulnerability, please report it to security@matrix.org**
+**If you've found a security vulnerability with the Libera Chat _Matrix Bridge Service_, please report it to security@matrix.org**
+
+**If you've found a vulnerablility with the Libera Chat _IRC Network_, please contact Libera Chat staff by emailing [support@libera.chat](mailto:support@libera.chat) or by private messsaging an active staffer on IRC**
```

Comment:
nit but do we have a way to identify what an active staffer looks like (cloaks?)

Reply:
The /stats p command will show active staffers, and all Libera staff have a libera/staff/* cloak